public/src/pages/views/MemberView.tsx

```diff
@@ -158,12 +134,39 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
             `Warning! This action is irreversible!
             Do you want to reject the student?`,
         )) {
-            const result =
-             await this.props.courseMan.changeUserState(userRel.link, Enrollment.UserStatus.REJECTED);
-            if (result) {
-                userRel.link.setStatus(Enrollment.UserStatus.REJECTED);
+            let readyToDelete =
+             await this.props.courseMan.isEmptyRepo(this.props.course.getId(), userRel.user.getId(), 0);
+            if (!readyToDelete) {
+                readyToDelete = confirm(
+                    `Warning! User repository is not empty.
+                    Do you still want to reject the user?`
```

Comment:
vscode (the linter probably) tells me that it wants a `,` after the backtick.

Reply:
An alternative is to move the `);` up to this line.

Reply:
Done.